README.md

```diff
@@ -1,13 +1,46 @@
 Enable [Yambar] to show [Hyprland] workspaces.
 
+# Installation
+
+Once installed use the Yambar config further down in the README. You'll need
+to adjust the `script` path depending on your installation method.
+
+## Cargo
+
 Assuming you have [Rust installed][Rust], run:
 
 ```console
 $ cargo install yambar-hyprland-wses
 ```
 
-and then use the Yambar config further down in the README. You'll need
-to adjust the `script` path.
+Use `/home/yourusernamehere/.cargo/bin/yambar-hyprland-wses` as your script path
+
+## Arch Linux
+
+There is a community maintained package available in the AUR. Assuming you have
+[yay], run:
```

Comment:
We could also link to [paru](https://github.com/Morganamilo/paru) which is also written in Rust (like this is!). It's what I personally use. Maybe insert a generic mention here of "any AUR helper". Could also give the generic AUR install instructions:

```console
$ git clone https://aur.archlinux.org/yambar-hyprland-wses.git
$ cd yambar-hyprland-wses
$ makepkg -si
```

Reply:
Ah yes it does feel right for a Rust utility to mention a Rust AUR helper :)

I have added instructions for Paru and manual installation